### PR TITLE
Programmers updates P1 - P6

### PR DESCRIPTION
**Pull request by MichellanneLi**
Hello @thetuttingtutor !

### In this PR
- updates to language

- fixes some inaccurate tabbing that probably carried over from the Google Doc

### Success Criteria
- language updates add clarity for user

- correct tabbing is reflected in viewing readme file in browser

- examples make sense and illustrate how to successfully implement a11y reqs

### Please Note

I added two new labels to our github repo: "in progress" and "ready for review." 

I sometimes create PRs when my work is still in-progress in order to share open questions, etc., and the "in progress" label reflects that status. 

I change the label to "ready for review" when I'm done, so that the reviewer knows to take a look.

Hope that's clear, and I'm happy to adjust my processes as needed for the team!

Thanks!

### CODE REVIEW

File: content/programmers.md [hunk 1]
```diff
@@ -144,6 +146,21 @@ Make the purpose of all UI components, icons, and regions programmatically deter
...
+#####Text input has a label
+
+```html
+<label for=”dogsname”>What is your dog’s name?</label>
+<input type=”text” id=”dogsname” name=”dogsname”>
+```
+
+####Bad example
+#####Text input has a placeholder attribute, but not a label
+
```
Comment (MichellanneLi) [inline content/programmers.md]: @thetuttingtutor add placeholder to glossary

File: content/programmers.md [hunk 2]
```diff
@@ -166,7 +183,11 @@ Ensure the purpose of links can be determined from the link text alone, or from
 ### P6. Make the name, role, and value of all UI components programmatically determined
 >  I need to use my screen reader to control my apps, so it needs to understand the state of all my settings so that I can change them if needed.
 
-Make the name, role, and value of all UI components - including form elements, links, and components generated by scripts - programmatically determined. Notify users of any changes made to these items.
+When creating custom interface components - including form elements, links, and components generated by scripts - that don't use standard markup, the name, role, and value must be programmatically determined. Notify users of any changes made to these items.
+
```
Comment (MichellanneLi) [inline content/programmers.md]: @MichellanneLi use accordion example

**Comment by MichellanneLi** [comment]
@thetuttingtutor This is ready for review. Thanks!

File: content/programmers.md (hunk 1 above)
Comment (thetuttingtutor) [reply content/programmers.md]: added term and definition, feel free to edit

**Pull request merged**